docker-compose-generator/docker-fragments/opt-add-teos.yml

```diff
@@ -0,0 +1,34 @@
+version: "3"
+
+services:
+  bitcoin_teos:
+    restart: unless-stopped
+    image: kukks/teos:cb1ba56
+    environment:
+      API_BIND: "0.0.0.0"
+      API_PORT: 9814
+      BTC_NETWORK: ${NBITCOIN_NETWORK:-regtest}
+      BTC_RPC_CONNECT: "bitcoind"
+      BTC_RPC_PORT: 43782
+      BTC_RPC_USER: "teos"
+      BTC_RPC_PASSWORD: "teos"
+      BTC_FEED_CONNECT: "bitcoind"
+      BTC_FEED_PORT: 28332
+    volumes:
+      - "bitcoin_teos_datadir:/root/.teos"
+    ports:
+      - "9814"
```

Comment:
should be `export 9814:9814` unless there's any good reason to not map it to that port.

Reply:
afaik saying "9814" maps to "9814:9814"

Reply:
It was not, if not specifying `9814:9814` I was getting it mapped to `32877`